recipes/stats_arrays/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "stats_arrays" %}
+{% set version = "0.6.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/stats_arrays-{{ version }}.tar.gz
+  sha256: 184102119f5884b8b522584b1e14d3a0c1571f524104a6529ce991e0928417cb
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - numpy
+    - python >=3.6
+    - scipy
+
+test:
+  imports:
+    - stats_arrays
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://bitbucket.org/cmutel/stats_arrays
+  summary: Standard NumPy array interface for defining uncertain parameters
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file:
+    - LICENSE
+  doc_url: http://stats-arrays.readthedocs.io/en/latest/
+  dev_url: https://github.com/brightway-lca/stats_arrays
+
+extra:
+  recipe-maintainers:
+    - cmutel
```

Comment:
@cmutel, the last one to confirm (for now).  Are you happy to be a maintainer here, too?

Reply:
@xylar Yes, thank you.